Terminal.Gui/Views/Menu.cs

```diff
@@ -998,9 +1007,10 @@ void Selected (MenuItem item)
 		public event Action<MenuItem> MenuOpened;
 
 		/// <summary>
-		/// Raised when a menu is closing.
+		/// Raised when a menu is closing passing <see langword="true"/>
+		/// if it's closing a sub-menu, <see langword="false"/> otherwise.
 		/// </summary>
-		public event Action MenuClosing;
+		public event Action<bool> MenuClosing;
```

Comment:
Isn't changing the signature of an event a breaking change?

Reply:
That's was a recent event implementation and I think it will not having a bigger impact.

Reply:
If we are changing this to take a parameter should we make it `MenuClosingEventArgs` or `MenuEventArgs` or something? that would future proof it for later on if we want to add more info about the menu event that is happening e.g. a cancel property etc.

Reply:
Done! https://github.com/migueldeicaza/gui.cs/pull/1615/commits/a2351b818105a7e45a3791befb382116298289c0